### PR TITLE
adds release.md

### PR DESCRIPTION
**Pull request by hbcarlos**
Adds `release.md` file for instructions on releasing the package.

cc @jtpio

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+```bash
+git clean -fdx
+```
+
+## Releasing on PyPI
+
+Make sure the `dist/` folder is empty.
+
+1. If the JupyterLab extension has changed, make sure to bump the version number in `./packages/jupyterlab-gridstack/package.json`
```
Comment (jtpio) [inline RELEASE.md:22]: Looks like this step could be removed? (covered in the next one)

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+8. `twine upload dist/*`
+
+# Making a new release of jupyterlab-drawio
+
+The prebuilt extension is already packaged in the main Python package.
+
+However we also publish it to `npm` to:
+
+- let other third-party extensions depend on `jupyterlab-drawio`
+- let users install from source if they would like to
```
Comment (SylvainCorlay) [inline RELEASE.md:38]: Or bundle the in the extension into the JupyterLab application.

File: RELEASE.md (hunk 1 above)
Comment (hbcarlos) [reply RELEASE.md:22]: yep

File: RELEASE.md (hunk 2 above)
Comment (hbcarlos) [reply RELEASE.md:38]: Do you mean adding a new point to specify that the extension could be bundled into the JupyterLab app?

Comment (SylvainCorlay) [reply RELEASE.md:37]: yep. btw, I there is a typo below (bundled instead of bundle).

Comment (hbcarlos) [reply RELEASE.md:37]: Thanks! changed.

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- bundled the extension into the JupyterLab app
```
Comment (jtpio) [inline RELEASE.md:38]:
```suggestion
- bundle the extension into the JupyterLab app
```

**Review by jtpio — APPROVED** [review]
Thanks!

**Pull request merged**